package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "tslint-config-shopify",
-  "version": "1.0.1",
+  "version": "0.0.0-semantically-released",
```

Comment:
As discussed in Slack, `semantic-release` is a bigger discussion than this PR, so it should be removed for now.